Dikubot/DataLayer/Database/Global/Facebook/FacebookModels.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
+
+namespace Dikubot.DataLayer.Database.Global.Facebook;
+
+
+public class FacebookPage : MainModel
+{
+    
+    public FBPages Item { get; set; }
```

Comment:
FacebookPage only contains a FBPages object. Why not make it 1 object instead of 2 different?

Reply:
Since the FacebookPage have ID as GUID, an FBPage have an ID as string/numbers